### ISSUE TITLE
GM8 / GMS Port

### ISSUE THREAD

**Issue by FoxFore32x** [basiccube/PizzaTower2000#1]
is it possible to adapt this project into Game Maker 8 and Game Maker Studio? since I'm interested in seeing being ported to more platforms like PSP, PS3, PS-Vita, etc.

**Comment by basiccube** [comment]
Unfortunately, from some quick testing, probably not. In GM8 an error occurs during sprite loading that is so unknown that it literally provides no info as to what the issue is and it crashes as a result. And if you remove the sprite loading functionality (which, I really don't think would work anyway on other platforms), it still doesn't work due to something else that is causing it to crash.
I also did test it in GameMaker-HTML5, due to it still being somewhat similar to 8, and while it does provide slightly more information about the error, it still isn't in any way helpful.

As for Studio, absolutely not as it makes use of functions such as execute_file and sound_add for the room system and audio, which don't exist since Studio. I doubt it would run on systems like the PSP anyway, considering memory requirements and such.

**Comment by FoxFore32x** [comment]
makes sense, i've tried doing it myself, but also one problem is that it needs to load external files like sounds and levels. would be cool to see it at some point tho, but yeah, porting a game maker 6 game to 8 is not easy.

btw i mentioned PSP because of ChovyGM, but i realized that the game probably is too heavy for the console to handle xd
